source/reference/verification/count.txt

```diff
@@ -0,0 +1,84 @@
+.. _c2c-verify-doc-counts:
+
+===========================
+Verify with Document Counts
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The most basic method of verifying sync is to compare the number of documents
```

Comment:
Can we single source this text with the concept page? Same question applies to the intros for other pages.

Reply:
I think it works better without.  The text only appears in two places and it's not always the same text.